changelog.md

```diff
@@ -46,6 +51,10 @@
   * jinja2 from 3.0.2 to 3.1.3 (#7107, #7109)
   * peewee from 3.15.0 to 3.17.1 (#7112)
   * diffoscope from 236 to 256 (#7125)
+  * pillow from 10.0.1 to 10.2.0 (#7107)
+  * semgrep from 0.98.0 to 1.57.0 (#7107)
+  * molecule from 3.0.2.1 to 4.0.4 (#7116)
```

Comment:
The molecule update was undone by #7130 so we probably don't need to include it or molecule-vagrant.

Reply:
Fixed!